README.md

```diff
@@ -17,17 +18,22 @@ Based on the [Native Terraform Module](https://www.terraform.io/docs/providers/a
 ## Example Usage
 ```hcl-terraform
 module "azure-media-services" {
-  source = "git@github.com:hmcts/cnp-module-azure-media-services"
-  location = "uksouth"
-  env = "prod"
-  product = "dm"
-  account_tier = "Premium"
+  source                   = "git@github.com:hmcts/cnp-module-azure-media-services"
+  location                 = "${var.location}"
+  env                      = "${var.env}"
+  tags                     = "${var.tags}"
```

Comment:
it's normally called common_tags when passed in https://github.com/hmcts/cnp-module-postgres#configuration

```suggestion
  common_tags                     = "${var.common_tags}"
```
(will need to readjust formatting)